src/screens/Profile.jsx

```diff
@@ -1,18 +1,21 @@
 import React from "react";
-
 import Card from "../components/Card";
 import styles from "./Profile.module.css";
-import { labs } from "../mocks";
 import Tags from "../components/Tags";
+import { useFetch, API_Company } from "../utility";
+import Loader from "../components/Loader";
 
 const Profile = (props) => {
-  const lab = labs.find((lab) => lab.id === parseInt(props.match.params.id));
+  const [data, loading] = useFetch(API_Company);
+  const lab = data.find((lab) => lab.id === parseInt(props.match.params.id));
```

Comment:
The show endpoint (`/labs/:id`) might return more information than presented on the index endpoint (`/labs`). When displaying information about a specific resource, you should use the former to ensure all data is fetched!